### DISCUSSION TITLE
Issues with `requirements.txt´

### DISCUSSION THREAD

**Discussion by Fellowcraftengage** [neo-project/neo#2850] [General]
Hey team, 

Please bare with me ;) I´m new to blockchain dev. I´m trying to setting up a NEO Node on AWS Ubuntu. I´ve done several steps but I´m hitting a wall. I´ve tried to get the requirements.txt via `wget https://raw.githubusercontent.com/neo-project/neo/master/requirements.txt´

I got this message when I tried to fetch it (see below) (also see attached screenshot for reference)

_"https://raw.githubusercontent.com/neo-project/neo/master/requirements.txt
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.110.133, 185.199.111.133, 185.199.108.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.110.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-03-13 15:11:00 ERROR 404: Not Found."_

It seems that the link to the requirements.txt file is not working properly.

My question is where do I find this requirements.txt document? 

Looking forward your quick response 

Kind regards,
Dave

<img width="1026" alt="Screenshot 2023-03-13 at 17 29 42" src="https://user-images.githubusercontent.com/90555571/224765470-5f9298b3-7a0d-45a9-8a19-c5d5738ed378.png">

**Comment by cschuchardt88** [comment]
What guide are you following? Normally people just [download](https://github.com/neo-project/neo-node/releases/tag/v3.6.0) and run the executable on a machine or use the "SCREEN" terminal program in linux to keep the application running.
